Mathlib/Topology/Constructions.lean

```diff
@@ -1747,6 +1747,13 @@ theorem continuous_uLift_down [TopologicalSpace X] : Continuous (ULift.down : UL
 theorem continuous_uLift_up [TopologicalSpace X] : Continuous (ULift.up : X → ULift.{v, u} X) :=
```

Comment:
Out of scope for this PR, but: @YaelDillies, am I right in saying the naming convention here (unfortunately) says this should be `continuous_uliftUp`?

Reply:
Yes, that's what the naming convention says